### ISSUE TITLE
does the project is outdated?

### ISSUE THREAD

**Issue by successismyonlyway** [angelozerr/vscode-xml-maven#8]
In the vscode, I can't find It in extention market

**Comment by angelozerr** [comment]
This project was just a POC to show the capability to extend lemminx the xml languge server used in vscode xml

The main idea is to consume lemminx maven an extension of lemminx in vscode-maven 
https://github.com/microsoft/vscode-maven/issues/47#issuecomment-659594891

**Comment by successismyonlyway** [comment]
> This project was just a POC to show the capability to extend lemminx the xml languge server used in vscode xml
> 
> The main idea is to consume lemminx maven an extension of lemminx in vscode-maven [microsoft/vscode-maven#47 (comment)](https://github.com/microsoft/vscode-maven/issues/47#issuecomment-659594891)

but it's so slow, that's IntelliJ Idea's profits

**Comment by angelozerr** [comment]
If you wish those features please add a comment in the vscode maven issue

**Comment by pbodnar** [comment]
> This project was just a POC to show the capability to extend lemminx the xml languge server used in vscode xml

Yeah, but this PoC is still referenced from the README of https://github.com/eclipse/lemminx-maven, which surely confuses lots of people. But personally, I wouldn't also know what to newly write into the README instead, now that it is clear that getting `lemminx-maven` into VS Code ([vscode-xml](https://github.com/redhat-developer/vscode-xml)) is not an easy process as might seem from the various texts to-date... :P
